gammapy/spectrum/tests/test_flux_point.py

```diff
@@ -79,6 +82,7 @@ def model(x):
 
 
 @requires_dependency('scipy')
+@pytest.mark.xfail(reason='I dont understand why we support this (joleroi)')
```

Comment:
The remaining test failure is also related to this. Apparently the original version supported the scenario that one energy bin is given, and the flux has a more complex shape

```
ipdb> type(energy_min)
<type 'int'>
ipdb> type(energy_max)
<type 'int'>
ipdb> int_flux.shape
(6, 6)
ipdb> 
```

Do you remember which use case(s) this was for?